src/composables/balances.ts

```diff
@@ -45,6 +47,21 @@ export function useBalances({ store }: IDefaultComposableOptions) {
       .toFixed(),
   );
 
+  const balancesCurrenciesTotal = computed(
+    () => {
+      const result = Object.entries(balances.value)
+        .reduce((total, [accountAddress, accountBalance]) => {
+          const selectedAccount = accounts.value.find((acc) => acc.address === accountAddress);
+          const currencyRate = selectedAccount?.protocol
```

Comment:
Protocol is required for each accounts so there is no need to check if it exists.